Packs/PrismaCloud/Layouts/layoutscontainer-Azure_Storage_Misconfiguration.json

```
@@ -0,0 +1,642 @@
+{
+    "id": "Azure Storage Misconfiguration Incident",
```

```suggestion
    "id": "Azure Storage Misconfiguration",
```
ID of layout should equal to the incident type it is related to